lib/taiko.js

```diff
@@ -381,6 +381,25 @@ module.exports.title = async () => {
     return result.result.value;
 };
 
+/**
+ * Sets a cookie with the given cookie data; may overwrite equivalent cookie if they exist.
+ *
+ * @example
+ * setCookie("CSRFToken","csrfToken", {url: "http://the-internet.herokuapp.com"})
```

Comment:
Can you please add an example with domain too.

Reply:
Done